backend/src/test/java/com/woowacourse/levellog/acceptance/AcceptanceTest.java

```diff
@@ -55,7 +57,20 @@ abstract class AcceptanceTest {
      */
     @BeforeEach
     public void setUp(final RestDocumentationContextProvider contextProvider) {
+        setRestAssuredPort();
+        setRestDocsSpec(contextProvider);
+    }
+
+    @AfterEach
+    public void claenDatabase() {
+        databaseCleaner.clean();
+    }
```

Comment:
```suggestion
    @AfterEach
    public void cleanDatabase() {
        databaseCleaner.clean();
    }
```

---

**(참고)**
`@AfterEach` 메서드 이름은 `tearDown`을 많이 쓰나봅니다.
![스크린샷 2022-08-10 오전 1 55 54](https://user-images.githubusercontent.com/68512686/183715828-e63c9ad6-6f4a-4c25-9ef7-c48acb48d796.png)